scripts/new_project.kts

```diff
@@ -59,7 +59,13 @@ object NewProject {
         get() = rootPath + projectFolderName
 
     private val rootPath: String
-        get() = System.getProperty("user.dir").replace("scripts", "")
+        get() = System.getProperty("user.dir").let { userDir ->
+            if (userDir.endsWith("/scripts")) {
```

Comment:
Let's use `SEPARATOR_SLASH`:
```suggestion
            if (userDir.endsWith("${SEPARATOR_SLASH}scripts")) {
```

Reply:
@Wadeewee I think we agreed to use `File.separator` to handle the issue with Window platform that the separator will be `\` instead of `/`. 😅 And we already define `private val fileSeparator = File.separator` in [line 34 ](https://github.com/nimblehq/android-templates/pull/287/files#diff-7eb696319d9f663e727291b14010369b88a196561c3982f204666a5f53df43b9R34), so `SEPARATOR_SLASH` might be redundant in this case. Please correct me if I'm wrong 🙇

Reply:
Oh, yeah @manh-t 👍  Thank for reminding me 🙏 Then we can use `fileSeparator` instead of `SEPARATOR_SLASH` cc @ryan-conway